specification/serialconsole/resource-manager/Microsoft.SerialConsole/preview/2018-07-01-preview/examples/EnableConsoleExample.json

```diff
@@ -0,0 +1,13 @@
+{
+    "parameters": {
+        "api-version": "2018-07-01-preview",
```

Comment:
According to spec, the `parameters` parameter should be present.

Reply:
@sergey-shandar I'm not seeing a `parameters` parameter within the spec (line 119 in latest commit). What are you seeing that is telling you that `parameters` should be present?

Reply:
It was there https://github.com/Azure/azure-rest-api-specs/blob/88b7d7da3c4996065197f947ac19650b6d3c62dd/specification/serialconsole/resource-manager/Microsoft.SerialConsole/preview/2018-07-01-preview/serialconsole.json#L82